torch/csrc/api/include/torch/nn/modules/sequential.h

```diff
@@ -35,8 +35,18 @@ class SequentialImpl : public Cloneable<SequentialImpl> {
     push_back(std::forward<Modules>(modules)...);
   }
 
-  /// reset() is empty for `Sequential`, since it does not have parameter of its
-  /// own.
+  /// Special cloning function for `Sequential` because it does not use
+  /// `reset()`.
+  std::shared_ptr<Module> clone() const override {
```

Comment:
nit (maybe?): I think the PyTorch code style requires implementations to be at the bottom of files when they are in a header.

Reply:
There's a few other places where this happens, if you want to do this in a different PR that's fine, definitely will make the big headers a lot more readable.